client/packages/common/src/intl/strings/IntlStrings.ts

```diff
@@ -8,7 +8,7 @@ export { UseTranslationResponse } from 'react-i18next';
 
 export interface TypedTFunction<Keys> {
   // basic usage
-  (key: Keys, options?: TOptions<Record<string, unknown>>): string;
+  (key: Keys, options?: TOptions<Record<string, unknown>> | string): string;
```

Comment:
This was incorrectly typed before. There is code to handle the string case, so it should be defined here too.